string name
string team
int32  aruco_name
---
